src/tasks/deploy-utils.js

```diff
@@ -68,7 +68,17 @@ function validateId(settings) {
  */
 
 gulp.task('validate:id', () => {
-  const file = fs.readFileSync(config.tkConfig, 'utf8');
+  let file;
```

Comment:
Are we cool with a 40 line function? Could this be cleaned up while touching this code?